gpiozero/output_devices.py

```diff
@@ -1528,6 +1538,13 @@ def __init__(
             ),
             pin_factory=pin_factory
         )
+        
+        if PiGPIOFactory is None or not isinstance(self.pin_factory, PiGPIOFactory):
+            warnings.warn(PWMSoftwareFallback(
+                'To reduce servo jitter, use the pigpio pin factory.'
+                'See https://gpiozero.readthedocs.io/en/stable/api_output.html#servo for more info'
```

Comment:
It seems that the documentation being pointed to doesn't mention PiGpio at all? Maybe this PR should also update the associated documentation?

Reply:
A note was added to the Servo docstring in this PR :)